Mathlib/LinearAlgebra/Dimension.lean

```diff
@@ -314,7 +314,15 @@ theorem LinearIndependent.set_finite_of_isNoetherian [IsNoetherian R M] {s : Set
 -- One might hope that a finite spanning set implies that any linearly independent set is finite.
 -- While this is true over a division ring
 -- (simply because any linearly independent set can be extended to a basis),
--- I'm not certain what more general statements are possible.
+-- or over a ring satisfying the strong rank condition
+-- (which covers all commutative rings; see `LinearIndependent.finite_of_le_span_finite`).
+-- this is not true in general.
```

Comment:
I think I should have said "or more generally over a ring satisfying ..." and the second line should end in a comma. Maybe @erdOne can fix these in #9300?